pos_receipt_hide_user/static/src/js/order_receipt.js

```diff
@@ -0,0 +1,43 @@
+odoo.define("pos_receipt_hide_user.OrderReceipt", function (require) {
+    "use strict";
+
+    const OrderReceipt = require("point_of_sale.OrderReceipt");
+    const Registries = require("point_of_sale.Registries");
+
+    const HideUserOrderReceipt = (OriginalOrderReceipt) =>
+        class extends OriginalOrderReceipt {
+            constructor() {
+                super(...arguments);
+                this.hide_option_from_config = this.env.pos.config.hide_user_option;
+            }
+
+            get HideUserOption() {
+                return this.hide_option_from_config;
+            }
+
+            GetTrig(name) {
+                const split_name = name.trim().split(/\s+/);
+
+                if (split_name === 0) return "";
+
+                const firstWord = split_name[0];
+                const firstLetters = firstWord.slice(0, 1);
+
+                const lastWords = split_name[split_name.length - 1];
+
+                const lastLetters =
+                    lastWords.length > 1 ? lastWords.slice(0, 2) : lastWords;
+
+                return lastLetters + firstLetters;
```

Comment:
Take the name of cashier is "Mike" for example, your method will return the value of its trigram is "MiM". i think it should be "Mik", @nilshamerlinck what do you think?